### PR TITLE
Testcase

### PR DESCRIPTION
**Pull request by 0b11001111**
I modified the test case a little in order to simplify my tests in pymontecarlo-reconstruction.

### CODE REVIEW

File: pymontecarlo/testcase.py [hunk 1]
```diff
@@ -124,21 +125,37 @@ def create_basic_photonintensityresult(self):
...
+        b.add_kratio((29, 'Kb5II'), 0.5, 1.0)
+        b.add_kratio((29, 'Ll'), 2.0, 1.0)
+        return b.build()
+
     def create_basic_simulation(self):
         options = self.create_basic_options()
 
         results = []
         results.append(self.create_basic_photonintensityresult())
+        results.append(self.create_basic_kratioresult())
```
Comment (ppinard) [inline pymontecarlo/testcase.py:145]: Why do you add two KRatioResult?

**Review by ppinard — CHANGES_REQUESTED** [review]
Everything else looks good

**Comment by ppinard** [comment]
Could you also fix the broken tests?

File: pymontecarlo/testcase.py (hunk 1 above)
Comment (0b11001111) [reply pymontecarlo/testcase.py:145]: This is what I've observed in real samples - two k-ratio results in one simulation. I was wondering too if that makes sense. However, it won't affect my code if one is removed.

**Review by ppinard — APPROVED** [review]

**Pull request merged**